### DISCUSSION TITLE
incorrect display (blurry) at a specific screen resolution

### DISCUSSION THREAD

**Discussion by wolowizard8** [antiden/macOS-cursors-for-Windows#51] [General]
at 1680x1050 (100%) resolution, most of the cursors are not clear
p.s. really like what you do. I hope cursors for this resolution will be available someday!
Thank you a lot!

this set is displayed most clearly for me, but it is not complete and not so considered
[El Capitan.zip](https://github.com/antiden/macOS-cursors-for-Windows/files/9006310/El.Capitan.zip)

**Comment by antiden** [comment]
@wolowizard8 Hi, thx, but my cursors only for 4K displays. I don't have screen with resolution 1680x1050. So I can't really help.

**Reply by wolowizard8** [reply]
ahh i get it, okey. Thanks anyway, good luck in all your endeavors!
